packages/fluent/scss/list/_layout.scss

```diff
@@ -101,9 +101,7 @@
         align-content: center;
         gap: var( --kendo-icon-spacing, .5rem );
         position: relative;
-        transition-property: color, background-color, outline-color, box-shadow;
-        transition-duration: 200ms;
-        transition-timing-function: ease;
+        @include kendo-motion(short-4, ease, color, background-color, outline-color, box-shadow);
```

Comment:
In the [motion easings list ](https://github.com/telerik/kendo-themes/pull/4779/commits/6af3b3b826d5c098967460b3d954abed183e5946#diff-2fbd99b1fa8fb0dd6af4da97661b0d7b5c4270893c91fa6ec813c8488d507e20R23)is missing ease - please change it with the correct one